rust/sbp/src/de.rs

```diff
@@ -303,11 +306,14 @@ impl Frame {
 
     pub fn to_sbp(&self) -> Result<Sbp, Error> {
```

Comment:
Can probably change this to `Result<Sbp, CrcError>`, I think that's the only thing that can go wrong now